specification/servicenetworking/ServiceNetworking.Management/main.tsp

```diff
@@ -14,9 +15,9 @@ namespace Microsoft.ServiceNetworking;
 
 /** Api versions */
 enum Versions {
-  /** 2023-11-01 preview version */
+  /** 2024-05-01 preview version */
   @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
-  `2023-11-01`,
```

Comment:
>`2023-11-01`, [](http://example.com/codeflow?start=2&length=13)

Why is the stable version being removed? Is this intentional?

Reply:
pl book an ARM API review office hours slot to discuss further. aka.ms\armofficehoursinfo

Reply:
@rkmanda This version will still be used. I couldn't find any documentation on how to implement multiple versions. But I can try keeping both versions in the tsp.

Reply:
@rkmanda

Reply:
I've reverted this change. @rkmanda